Your_second_ASDF_converter.ipynb

```diff
@@ -0,0 +1,1142 @@
+    "the schemas, a separate one for the converters (again, so that\n",
+    "the schemas are not tied directly to Python; but note there\n",
+    "will be a small amount of code in the schema package for Python,\n",
+    "and we expect that if other languages are used, code for those\n",
+    "may appear there as well).\n",
+    "\n",
+    "A utility has been developed so that the schema package is \n",
+    "fairly easily generated and this tutorial will use that utility.\n",
+    "The structure of the converter package will be very similar to\n",
+    "that of the first tutorial.\n",
+    "\n",
+    "Required Software installed\n",
+    "--------------------------------\n",
+    "\n",
+    "- numpy\n",
+    "- asdf v2.8 or higher\n",
+    "\n",
+    "The following is recommended when creating your own package but \n",
+    "not directly needed for this tutorial\n",
+    "\n",
+    "- cookiecutter (do a `pip install cookiecutter`)\n",
+    "\n",
+    "Software uninstall\n",
+    "---------------------\n",
+    "\n",
+    "If you did the first tutorial (Your_first_ASDF_converter), in a terminal window \n",
+    "type `pip uninstall mfconverter` since this tutorial will reuse the same \n",
+    "photoID class and we don't want confusion between the two converters.\n",
+    "\n",
+    "Create schema package\n",
+    "----------------------------\n",
+    "\n",
+    "First ensure you are in the directory that you want to create both packages in, by default it will be your home directory below, but change it to what you want before executing."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bb0c1b6e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cd ~/converter_tutorial\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "067eed9a",
+   "metadata": {},
+   "source": [
+    "The following utility will build a schema package. It will\n",
+    "prompt for answers to a series of questions. Here is an \n",
+    "explanation of what is expected for these questions. **You must\n",
+    "run this utility from a terminal window.** Type the command as\n",
+    "shown in this directory and provide the answers as shown below.\n",
+    "If this is rerun you may get an initial prompt mentioning the\n",
+    "template has been downloaded before; just type return to accept\n",
+    "the default `yes`\n",
+    "\n",
+    "`\n",
+    "cookiecutter gh:asdf-format/schemas-package-template\n",
+    "`\n",
+    "\n",
+    "`\n",
+    "package_name [asdf-example-schemas]: myschemas\n",
+    "`\n",
+    "\n",
+    "This will be the directory name it will go in.\n",
+    "\n",
+    "`\n",
+    "module_name [myschemas]: \n",
+    "`\n",
+    "\n",
+    "Accept the default (providing a different answer will create a\n",
+    "different module name within the package\n",
+    "\n",
+    "`\n",
+    "Short_description [mfconverter]: \"my first schemas” \n",
```

Comment:
Are the quotes necessary here?

Reply:
I'm not sure, but since you asked, I removed them.